pkg/controller/migplan/migplan_controller.go

```diff
@@ -362,3 +371,16 @@ func (r *ReconcileMigPlan) planSuspended(plan *migapi.MigPlan) error {
 
 	return nil
 }
+
+// Update Status.ExcludedResources based on
+// "EXCLUDE_RESOURCES" environment variable
+func (r *ReconcileMigPlan) ensureExcludedResourceList(plan *migapi.MigPlan) error {
+	var excludedResources []string
+	resourcesEnv := os.Getenv("EXCLUDE_RESOURCES")
```

Comment:
All of the controller settings (and environment varables) are encapsulated in the _settings_ package.  

Let's add `ExcludedResources []string` [here](https://github.com/konveyor/mig-controller/blob/master/pkg/settings/plan.go#L14).

Which is loaded are parsed into the list [here](https://github.com/konveyor/mig-controller/blob/master/pkg/settings/plan.go#L21)

And used:
```
excludedResources := Settings.Plan.ExcludedResources
```